modules/migration-known-issues.adoc

```diff
@@ -28,4 +28,6 @@ These annotations preserve the UID range, ensuring that the containers retain th
 
 ifdef::migrating-3-4[]
 * In the current release (CAM 1.2), you cannot migrate from {product-title} 3.7 to 4.4 because certain API `GroupVersionKinds` (GVKs) that are used by the source cluster are deprecated. You can manually update the GVKs after migration. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1817251[*BZ#1817251*])
+
+* If you cannot install CAM 1.2 on an {product-title} 3 cluster, download the current `operator.yml` file, which fixes this problem. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1843059[*BZ#1843059*])
```

Comment:
This LGTM, though I'd ask whether you'd be able to add a link to the current `operator.yml` file? I looked at the BZ and didn't see an obvious link that would stand out to users.

Reply:
Good idea!

Reply:
On second thought,the operator.yml has to be downloaded with Podman. It's not a simple link. The user will be in the process of installing CAM, so they will have just downloaded the file. The Known Issue is to alert users who might have gotten the bad image by mistake, during the period between 1.2 release and this fix.